CONTRIBUTING.md

```diff
@@ -58,6 +58,20 @@ formatted, these will appear directly in the changelog.
 ### Source Modifications
 
 Modifications are primarily in `projects/common` project. This is a common library that is shared between the deployment
-environments. The on-prem project is the default deployment environment and does not require any external modification to run.
+environments. The on-prem project is the default deployment environment and does not require any external modification
+to run.
 
-If running the cloud deployment environment, a TIBCO Cloud Account will be needed to access UI.
\ No newline at end of file
+If running the cloud deployment environment, a TIBCO Cloud Account will be needed to access the UI.
+
+### Adding New Components or Schematics to the Common Library
+
+Components or schematics need to be added using `ng g <schematic> <name> <options> --skip-import` in the root of the project.
+An example of creating a new common component would be `ng g c newComponent --project common --skip-import`.
+The `skip-import` is needed because there is more than one module that matches that name in the project.
```

Comment:
We might as well be specific with the command so it is very clear even to people less familiar with the tools.

```suggestion
Components or schematics need to be added using `ng generate <schematic> <name> <options> --skip-import` in the root of the project.
An example of creating a new common component would be `ng generate component newComponent --project common --skip-import`.
The `skip-import` is needed because there is more than one module that matches that name in the project.
```

Instead of using `--skip-import`, should we use `--module xxx`? Would that result in fewer user errors?

Reply:
Let me try the `--module` approach. And I have Mark All in One lint and the markdownlint extensions enabled. Oddly they did not show any issues when I committed.

Reply:
Neat, I got it to work. I'll update the contributions accordingly.